plugin/plugin.go

```diff
@@ -154,8 +154,22 @@ func (p *plugin) generateRegexVars(file *generator.FileDescriptor, message *gene
 	ccTypeName := generator.CamelCaseSlice(message.TypeName())
 	for _, field := range message.Field {
 		validator := getFieldValidatorIfAny(field)
-		if validator != nil && validator.Regex != nil {
+		if validator != nil && (validator.Regex != nil || validator.UuidVer != nil) {
 			fieldName := p.GetOneOfFieldName(message, field)
+			if validator.UuidVer != nil {
+				if uuid, err := getUUIDRegex(int(validator.GetUuidVer())); err != nil {
+					fmt.Fprintf(
+						os.Stderr,
+						"WARNING: field %v.%v error %s.\n",
+						ccTypeName,
+						fieldName,
+						err,
+					)
```

Comment:
No need to split all of this, can be on a single line as there's no "line-length limit" in Go best-practices.

Reply:
fixed